Developer Training/docs/command_to_make_pdf.txt

```diff
@@ -0,0 +1 @@
+pandoc .\developer_guide.md -f markdown --listings -H .\developer_guide.tex -s --toc -o Developer_Guide.pdf
```

Comment:
this file can probably be removed (there is a github actions job that does the conversion)